ledger/narwhal/transmission-id/src/string.rs

```diff
@@ -43,8 +52,8 @@ impl<N: Network> Display for TransmissionID<N> {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         match self {
             Self::Ratification => write!(f, "ratification"),
-            Self::Solution(id) => write!(f, "{}", id),
-            Self::Transaction(id) => write!(f, "{}", id),
+            Self::Solution(id, checksum) => write!(f, "{}.{}", id, checksum),
```

Comment:
it may be worth considering encoding the checksum as padded base16 or bech32 without a human-readable-part rather than base10